CHANGELOG.md

```diff
@@ -13,7 +13,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixes
 
--
+- Fix metrics port during operator startup.
+  The metrics port configuration was being set to a default of 8080 due to an
+  unused options setup variable, this is corrected to be hard-coded to 8383 as
+  required by OCP monitoring. Fixes [OCP-3097](https://issues.redhat.com/browse/OCPBUGS-3097).
```

Comment:
Do we also want to note that the operator is watching the proper namespace, allowing for more efficiency at runtime?

Reply:
updated